docs/cli/index.md

```diff
@@ -22,7 +22,7 @@ A developer tool for building Cosmos SDK blockchains
 
 **SEE ALSO**
 
-* [starport chain](#starport-chain)	 - Build, initialize, and start a blockchain
+* [starport chain](#starport-chain)	 - Build, initialize, and start a blockchain in development
```

Comment:
Anything under docs/cli is auto generated from the command definitions under cmd/.

Can we make these changes inside the cmd/ instead?

For ex, for this one we need to update: https://github.com/tendermint/starport/blob/cc9e999/starport/cmd/chain.go#L10